app/models/user.rb

```diff
@@ -223,8 +223,8 @@ def block!
       update_attribute(:email, "security+locked-#{SecureRandom.hex(4)}-#{id}-#{handle}@rubygems.org")
       confirm_email!
       disable_mfa!
+      update_attribute(:password, SecureRandom.alphanumeric)
```

Comment:
Any reason to not use `update_columns` for all update attributes at once?

Reply:
I though I had, but not really, I'm changing update to update_columns

Thanks!

Reply:
Now I'm not sure if you meant all other attributes as email related and `mfa_*` fields. IMO, all can be modified on this block, but not sure if it was planned as it is to don't repeat logic.

Reply:
The password field need to be updated with `update_attribute` or `update`, `update!` because it's not a "real" column on the database, `update_column`t triggers ...

```ruby
ActiveModel::MissingAttributeError: can't write unknown attribute `password`
```